grumblr/db/seeds.rb

```diff
@@ -1,7 +1,8 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+grumblrs = Grumblr.create([
```

Comment:
Your seeds need to clean up after themselves. If I had to guess, this is why your "seeds didn't go through". Just add `Grumblr.delete_all` to the line before this one. You also don't need to assign these to a variable (remove `grumblrs = `).